amplify/data/resource.ts

```diff
@@ -23,13 +23,15 @@ const TemplateStorageModel = {
   }),
 };
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const authPermission = (allow: any) =>
```

Comment:
please can we get the actual type rather than any? it should be available somewhere in one of the amplify packages

Reply:
Looks like the type is not available to import, this seems a reasonable compromise to prevent duplicating the logic in multiple places given that this code is temporary.